### ISSUE TITLE
IOS平台加入热更新后审核被拒

### ISSUE THREAD

**Issue by tj123678** [Tencent/xLua#610]
嗨，请问一下，我们准备尝试在ios平台上使用Xlua热更新。但是仅仅只是在导入XLua库后，发布ios平台，结果就审核被拒，请问该如何解决了，ios平台能否使用热更新了。
下面是被拒信息：

Guideline 2.1 - Information Needed

This type of app has been identified as one that may violate one or more of the following App Store Review Guidelines. Specifically, these types of apps often:

1.1.6 - Include false information, features, or misleading metadata
2.3.0 - Undergo significant concept changes after approval
2.3.1 - Have hidden or undocumented features, including hidden "switches" that redirect to a gambling or lottery website
3.1.1 - Use payment mechanisms other than in-app purchase to unlock features or functionality in the app
3.2.1 - Do not come from the financial institution performing the loan services
4.3.0 - Are a duplicate of another app or are conspicuously similar to another app
5.2.1 - Were not submitted by the legal entity that owns and is responsible for offering any services provided by the app
5.2.3 - Facilitate illegal file sharing or include the ability to save, convert, or download media from third party sources without explicit authorization from those sources
5.3.4 - Do not have the necessary licensing and permissions for all the locations where the app is used

Before we can continue with our review, please confirm that this app does not violate any of the above guidelines. You may reply to this message in Resolution Center or the App Review Information section in App Store Connect to verify this app’s compliance. 

Given the tendency for apps of this type to violate the aforementioned guidelines, this review will take additional time. If at any time we discover that this app is in violation of these guidelines, the app will be rejected and removed from the App Store, and it may result in the termination of your Apple Developer Program account.

Should you choose to resubmit this app without confirming this app’s compliance, the next submission of this app will still require a longer review time. Additionally, this app will not be eligible for an expedited review until we have received your confirmation.

**Comment by yindongfei** [comment]
除了加入xlua的库，没有其他操作了么？

**Comment by tj123678** [comment]
有给自己写的两个类打了hotfix标签，其他没有操作了

**Comment by yindongfei** [comment]
除了xlua，没有引入其他库？

**Comment by tj123678** [comment]
没有耶

**Comment by yindongfei** [comment]
在加入xlua之前，是可以审过的？是机审挂了还是到人工了？

**Comment by tj123678** [comment]
加入Xlua之前是可以过的，这个应该是机审挂了。

**Comment by yindongfei** [comment]
可以考虑混淆一下xlua的dll试试。。。如果啥都没有变，只是被检测到使用了xlua的dll，就被拒了，问题就严重了，也许apple开始查lua热更新了？

**Comment by chexiongsheng** [comment]
我了解到的情况是最近机审的规则有变化。有部分游戏之前一直ok，但最近被拒，这些被拒的游戏有用了xlua，也有没用xlua的。也有游戏用xlua但是过了的。目前看两者没必然关系。

**Comment by chenqiu1988** [comment]
这个是2.1大礼包呀,不用管,怼回去就行了

**Comment by chenqiu1988** [comment]
和你是否用xlua没什么关系

**Comment by tj123678** [comment]
删除了xlua，重新提审就过了，可能最新苹果对热更新审核比较严
